public/app/core/services/keybindingSrv.ts

```diff
@@ -47,6 +47,7 @@ export class KeybindingSrv {
 
     this.bind('t t', () => toggleTheme(false));
     this.bind('t r', () => toggleTheme(true));
+    this.bind('t n', () => this.toggleNav());
```

Comment:
do you think maybe we should gate this behind _another_ feature flag, or just in development or something? Being something that's a pretty big change that we wouldn't want someone to do inadvertently?

Reply:
@joshhunt can gate it behind Grafana being in dev mode

Reply:
Yeah I think that’s sensible 😌